### PR TITLE
Team4 question10

### PR DESCRIPTION
**Pull request by schultzh**
This PR contains the answer to the question: "What is your team's software development and testing process for creating new features?".  The text is broken into subsections to ensure a clear process is presented.

### CODE REVIEW

**Review by MaddyKapfhammer — APPROVED** [review]

**Review by noorbuchi — APPROVED** [review]
Looks good!

**Review by ibarrezueta — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]
Looks good. For the future project, you guys can even come up with a branch name template, just like what you have here `feature/description-of-the-feature`

File: management.md [hunk 1]
```diff
@@ -32,6 +32,38 @@
...
+    GitHub Issue Tracker. This issue should outline the purpose of the new
+    feature and the steps needed for feature development. If this feature solves
+    any current issues open in the Issue Tracker, connect the issue numbers.
+    Next, assign the members of your development team and request feedback
+    from team members, Technical Leaders, and the course instructor.
+
+  - Create a Branch
+
+    The next step is to create a branch off of master with the following naming
+    convention: feature/description-of-the-feature. All the work for the new
```
Comment (enpuyou) [inline management.md]: Small issue: I would add "`" around feature/description-of-the-feature

Comment (schultzh) [reply management.md]: Got it! Thanks!

Comment (enpuyou) [reply management.md]: I actually meant to use the backtick, instead of the quotation mark, sorry for the confusion

**Pull request merged**